tests/test_functional_post_install.py

```diff
@@ -108,10 +115,20 @@ def test_rose_fileinstall_uses_suite_defines(tmp_path):
             # {file: content}
             {
                 'test/rose-suite.conf': 'opts=foo',
+                'test/opt/rose-suite-foo.conf': '',
+                'test/opt/rose-suite-bar.conf': '',
+                'test/opt/rose-suite-baz.conf': '',
                 'test/opt/rose-suite-cylc-install.conf':
                     '!opts=bar\n[env]\nBAR=1',
                 'ref/opt/rose-suite-cylc-install.conf':
                     '!opts=bar baz\n[env]\nBAR=2',
+                'ref/rose-suite.conf': '!opts=foo bar baz (cylc-install)',
+                'ref/opt/rose-suite-foo.conf': '',
+                'ref/opt/rose-suite-bar.conf': '',
+                'ref/opt/rose-suite-baz.conf': '',
+                'ref/log/18151210T0000Z-rose-suite.conf': (
+                    '!opts=bar baz\n\n[env]\nBAR=2\n'
+                ),
```

Comment:
I haven't understood why these are being added to the test

Reply:
The tests tests that the lists of files in the test and reference directory are the same: `assert titems == ritems` at around line 230.

Reply:
Ok, was mainly wondering why so many